docs/docs/components/vector-stores.mdx

```diff
@@ -26,7 +26,8 @@ The `Astra DB` initializes a vector store using Astra DB from records. It create
 - **Collection Indexing Policy:** Indexing policy for the collection.
 
 <Admonition type="note" title="Note">
-    Ensure you configure the necessary Astra DB token and API endpoint before starting.
+  Ensure you configure the necessary Astra DB token and API endpoint before
+  starting.
```

Comment:
Stray new line

Reply:
It's getting formatted due to pre commit hook which will run make format every time i commit.